src/__tests__/Login.test.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+import Login from "../components/Login";
+
+describe("login snapshot test", () => {
```

Comment:
You don't need two describes here and can place the snapshot test as the first test in the other describe